### ISSUE TITLE
Make App.launch faster by changing the calling parameter

### ISSUE THREAD

**Issue by txchen** [kasper/phoenix#211]
- Version: 2.6.1 (65)
- macOS: 10.13.4

I am currently using this app to toggle my apps: https://github.com/gbammc/Thor/

It's toggle code is simple: https://github.com/gbammc/Thor/blob/4f86a1768d33f7dbaf61182e2f10179a6166ec6a/Thor/ShortcutMonitor.swift#L17-L23

Now if I want to do the same with phoenix, after I call App.launch(name), I need to make another call to let the app focus, like this:

```
let app = App.launch('iTerm')
  if (app) {
    app.focus()
  }
```

Per my feeling (because I have not a good way to measure), toggling with Thor is faster than using Phoenix script. I think it is due to they are calling different API.

So, would it be okay to change the launch method (https://github.com/kasper/phoenix/blob/f02761eccd21033462efcf175e5d6b6d984f3cb8/Phoenix/PHApp.m#L44) to simply calling `https://developer.apple.com/documentation/appkit/nsworkspace/1531434-launchapplication`, or change the `options:NSWorkspaceLaunchWithoutActivation` to default, so that `app.focus()` can be avoided.

Or, maybe add a new method?

**Comment by txchen** [comment]
https://developer.apple.com/documentation/appkit/nsworkspace/1531434-launchapplication returns boolean, so App.launch(name) cannot use this because it will break scripts.

How about add a new API like: App.launchAndShow(appName)
The OC part just call `NSWorkspace.shared.launchApplication(appName)` and returns boolean.

**Comment by kasper** [comment]
Right! I think we can just add an options object as an additional argument for the `App#launch` function, i.e. `App.launch('Safari', { focus: true })`. It would still default to being false without any options.

**Comment by txchen** [comment]
I tried to make this change: 

```
--- a/Phoenix/PHApp.m
+++ b/Phoenix/PHApp.m
@@ -53,7 +53,7 @@ static NSString * const PHAppForceOptionKey = @"force";

     NSError *error;
     NSRunningApplication *app = [sharedWorkspace launchApplicationAtURL:[NSURL fileURLWithPath:appPath]
-                                                                options:NSWorkspaceLaunchWithoutActivation
+                                                                options:NSWorkspaceLaunchDefault
                                                           configuration:@{}
                                                                   error:&error];
     if (error) {
```

And I can confirm that with this change, opened App can show faster (about same speed of Thor, which uses `NSWorkspace.shared.launchApplication(appName)`)
Although NSWorkspace.shared.launchApplication(appName) looks even cheaper in my use case, I think `launchApplicationAtURL` with `options:NSWorkspaceLaunchDefault` is good enough.

**Comment by txchen** [comment]
PR created: https://github.com/kasper/phoenix/pull/212
Hope new build can be released so that I can reduce one opening app on my laptop, LOL.
